icicle-cpu/src/lifter/mod.rs

```diff
@@ -997,12 +1029,12 @@ impl BlockExit {
 }
 
 impl<T> pcode::PcodeDisplay<T> for BlockExit
-where
-    pcode::VarNode: pcode::PcodeDisplay<T>,
+    where
+        pcode::VarNode: pcode::PcodeDisplay<T>,
 {
     fn fmt(&self, f: &mut std::fmt::Formatter, ctx: &T) -> std::fmt::Result {
         match self {
-            Self::Jump { target: Target::Invalid } => f.write_str("invalid_instruction"),
+            Self::Jump { target: Target::Invalid(_, _) } => f.write_str("invalid_instruction"), // TODO: handle error specifically?
```

Comment:
Similar to above -- this is used for emulator debugging, so extra info could be could to print.